package.json

```diff
@@ -61,6 +61,7 @@
     "spectrum-generator": "^4.4.1"
   },
   "dependencies": {
+    "lodash": "^4.17.21",
```

Comment:
we should avoid lodash. it's a huge dependency

Reply:
So, the function seems better for you?

Reply:
yes